### PR TITLE
Eases dependency on httparty

### PR DESCRIPTION
**Pull request by josh-m-sharpe**

### CODE REVIEW

File: spec/dummy/log/development.log [hunk 1]
```diff
@@ -0,0 +1 @@
+DEPRECATION WARNING: ActiveSupport.halt_callback_chains_on_return_false= is deprecated and will be removed in Rails 5.2. (called from <top (required)> at /Users/jsharpe/rails_critical_css_server/spec/dummy/config/initializers/new_framework_defaults.rb:21)
```
Comment (wheeyls) [inline spec/dummy/log/development.log]: Did you mean to commit this file? I'm not sure what this is, but it looks like it should be removed and/or .gitignored?

**Comment by josh-m-sharpe** [comment]
@wheeyls fixed

File: lib/rails_critical_css_server/rewrite.rb [hunk 2]
```diff
@@ -48,13 +48,15 @@ def original_html
       @original_html = template.capture { block.call }
     end
 
+    def extracted_css_files
+      @extracted_css_files = ExtractCssFromHtml.call(original_html)
```
Comment (wheeyls) [inline lib/rails_critical_css_server/rewrite.rb]: Did you mean for this to be `||=`?

I'm on vacation til tomorrow, will try to get this merged when I come back.

**Comment by wheeyls** [comment]
Did you mean to close this PR?

**Pull request closed without merging**